src/components/SettingDialog.vue

```diff
@@ -884,6 +884,35 @@
                 >
                 </q-toggle>
               </q-card-actions>
+              <q-card-actions class="q-px-md q-py-none bg-surface">
+                <div>同じ読みのアクセント句内のパラメータの保持</div>
+                <div
+                  aria-label="テキスト変更時、同じ読みのアクセント句内のパラメータを保持します。"
+                >
+                  <q-icon name="help_outline" size="sm" class="help-hover-icon">
+                    <q-tooltip
+                      :delay="500"
+                      anchor="center right"
+                      self="center left"
+                      transition-show="jump-right"
+                      transition-hide="jump-left"
+                    >
+                      テキスト変更時、同じ読みのアクセント句内のパラメータを保持します。
+                    </q-tooltip>
+                  </q-icon>
+                </div>
+                <q-space />
+                <q-toggle
+                  :model-value="experimentalSetting.shouldKeepAudioParameter"
+                  @update:model-value="
+                    changeExperimentalSetting(
+                      'shouldKeepAudioParameter',
```

Comment:
```suggestion
                      'enableKeepAudioParameter',
```
`shuold`はあんまり使われていなさそうなので、他の実験的機能に合わせて`enable`のほうがいいかも...?

Reply:
動詞の場合shouldの方が正しい気がするんですよね（shouldApplyDefaultPresetOnVoiceChangedもある）

Reply:
横からすみません！
一応`enableKeepingAudioParameter`という名称も可能だとは思います。

気になったから意味的な違いをいろいろ考えてみてました。

* on/offの完全な切り替えする機能　→　「有効かどうか」を意味するenableのが合ってそう
* できる限りonまたはoffにする機能　→　「べき」を意味するshouldのが合ってそう

みたいな感じかもです。
（ちなみにChatGPT君の受け売りです･･･）
https://chat.openai.com/share/63cbb2ea-5b4d-4051-b83e-724269ce785d